recipes/recipes_emscripten/llvm/recipe.yaml

```diff
@@ -0,0 +1,29 @@
+context:
+  version: 16.0.6
+
+package:
+  name: llvm
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-{{ version }}.tar.gz
+  sha256: 56b2f75fdaa95ad5e477a246d3f0d164964ab066b4619a01836ef08e475ec9d5
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - '{{ compiler("c") }}'
+    - '{{ compiler("cxx") }}'
+    - cmake
+    - ninja
+
+about:
+  home: https://llvm.org
+  license: Apache-2.0
```

Comment:
```suggestion
  license: Apache 2.0 License with LLVM exceptions
```

Reply:
The license string on conda-forge is Apache-2.0 WITH LLVM-exception. Conda-forge does some checks on license string, so this may be the correct form.